src/WCSimPrimaryGeneratorAction.cc

```diff
@@ -90,8 +96,66 @@ WCSimPrimaryGeneratorAction::WCSimPrimaryGeneratorAction(
   useGunEvt    		= false;
   useLaserEvt  		= false;
   useGPSEvt    		= false;
+  useCosmics            = false;
   useRadioactiveEvt  	= false;
   useRadonEvt        	= false;
+
+  // Create the relevant histograms to generate muons
```

Comment:
You are right, the code was calling a single input file rather than a separate file for comics and vector data. I have fixed this see: 52e063abba1152a0a90cb1d0c7ab900b82f60f07.